x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -247,36 +247,47 @@ const getProductTypes = (
 export const cli = () => {
   run(
     async (context) => {
+      const environment = process.env.ENVIRONMENT ? `${process.env.ENVIRONMENT}` : 'qa';
+
+      const PROXY_URL = process.env.PROXY_URL ? process.env.PROXY_URL : undefined;
+      const PROXY_SECRET = process.env.PROXY_SECRET || undefined;
+      const PROXY_CLIENT_ID = process.env.PROXY_CLIENT_ID || undefined;
+
       // Checking if API key is either provided via env variable or in ~/.elastic.cloud.json
       // This works for either local executions or fallback in case proxy service is unavailable.
-      if (!process.env.CLOUD_QA_API_KEY && !getApiKeyFromElasticCloudJsonFile()) {
-        log.error('The API key for the environment needs to be provided with the env var API_KEY.');
+      if (!process.env.CLOUD_API_KEY && !getApiKeyFromElasticCloudJsonFile(environment)) {
```

Comment:
nit: `getApiKeyFromElasticCloudJsonFile(environment)` could be invoked once and the returned value reused twice instead of calling `getApiKeyFromElasticCloudJsonFile(environment)` twice.

Reply:
Fixed.